Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOAddressSummary.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import SwiftUI
+
+struct PPOAddressSummary: View {
+  let address: String
+
+  var body: some View {
+    HStack(alignment: .firstTextBaseline) {
+      // TODO: Localize
+      Text("Shipping address")
+        .lineLimit(nil)
+        .font(Font(Constants.labelFont))
+        .foregroundStyle(Color(Constants.labelColor))
+        .frame(width: Constants.labelWidth, alignment: .leading)
+        .multilineTextAlignment(.leading)
+
+      Text(self.address)
+        .lineLimit(nil)
+        .multilineTextAlignment(.leading)
+        .font(Font(Constants.addressFont))
+        .foregroundStyle(Color(Constants.addressColor))
+        .frame(maxWidth: .infinity, alignment: .leading)
+    }
+    .frame(maxWidth: .infinity)
+  }
+
+  private enum Constants {
+    static let labelWidth: CGFloat = 85
```

Comment:
What's the benefit of hardcoding the label width? I have a feeling we'd want it to be able to grow larger for larger font sizes (you're welcome to check with accessibility sizes in your simulator!) or in landscape/on iPad. Maybe calculate the width based on the screen size instead? (I think GeometryReader allows this but lmk if I'm wrong and I'd be happy to help brainstorm.)

Feel free to file a followup ticket (doesn't necessarily need to block launch) if you'd rather revisit layouts for large text later - we can follow up with UX.

Reply:
Injecting this value now to calculate the result from the parent card.